test/ProgressOnderwijsUtils.Drawing.Test/ProgressOnderwijsUtils.Drawing.Test.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <DebugType>full</DebugType>
+    <DebugSymbols>True</DebugSymbols>
+    <OutputType>library</OutputType>
+    <TargetFrameworks>net461</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="rainbow.jpg" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="ExpressionToCodeLib" version="2.5.1" />
+    <PackageReference Include="xunit" Version="2.3.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <PackageReference Include="JetBrains.Annotations" Version="11.1.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
+    <ProjectReference Include="..\..\src\ProgressOnderwijsUtils.Drawing\ProgressOnderwijsUtils.Drawing.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+</Project>
```

Comment:
En los test project is niet *noodzakelijk*.  Is het wenselijk?  Voordeel: minder interacties+dus complexiteit; mogelijk snellere build.  Nadeel: complexere test-run; mogelijk tragere build; mogelijk tragere test run.

Reply:
Nog een voordeel van een los testproject is dat we dan geen ifdefs meer nodig hebben. Bij elkaar instoppen lijkt me ook arbitrair aangezien er geen dependencies of zo zijn.